.aspire/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "appHostPath": "../src/eShop.AppHost/eShop.AppHost.csproj"
+}
```

Comment:
Assuming we don't want to checkin this file as it is likely just the result of running `aspire run`. If @captainsafia confirms I can remove.